verygoodsecurity/vgs-satellite
# [SAT-88] Dockerfile.

## Fixes [SAT-88](https://verygoodsecurity.atlassian.net/browse/SAT-88)

## Description of changes in release / Impact of release:
Dockerfile for Satellite.

Test plan:

1. _make image_
2. _docker run --rm -v $HOME/.vgs-satellite/:/data -p 8089:8089 -p 9098:9098 -p 9099:9099 -p 1234:1234 verygoodsecurity/vgs-satellite:1.0.1_
3. Visit http://localhost:1234 and make sure it works as expected.

### Is this a breaking change?

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
(insert text here)
